.github/actions/build-core/src/core-builder.ts

```diff
@@ -21,7 +21,7 @@ const COMMANDS: Commands = {
   gradle: [
     {
       cmd: gradleCmd,
-      args: ['createDistPrep'],
+      args: ['createDistPrep', '--stacktrace'],
```

Comment:
Maybe the `stacktrace` mode should be some parameter we can send from the actual GH workflow file.

Reply:
If there are no errors, there won't be stack traces generated.

Reply:
@victoralfaro-dotcms I am not sure if there is any reason not to add this parameter though.  I think it only impacts if we see the stack trace or not when there is an exception thrown by the core junit processing.   When there is no stacktrace shown we cannot diagnose and there is no further information other than re-running.  So adding stack trace all the time will not do anything when it runs correctly and will help diagnosis if it fails.